MyPerfectOnboardingApplication/Sources/MyPerfectOnboarding.Api.Services/ServicesBootstraper.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.Net.Http;
+using System.Web;
+using MyPerfectOnboarding.Contracts;
+using MyPerfectOnboarding.Contracts.Services.Location;
+using Unity;
+using Unity.Injection;
+using Unity.Lifetime;
+
+namespace MyPerfectOnboarding.Api.Services.Location
+{
+    public class ServicesBootstraper : IBootstraper
+    {
+        public IUnityContainer RegisterTypesTo(IUnityContainer container)
+            => container
+                .RegisterType<IUrlLocator, UrlLocator>(new HierarchicalLifetimeManager())
+                .RegisterType<HttpRequestMessage>(new HierarchicalLifetimeManager(), new InjectionFactory(unityContainer => HttpContext.Current.Items["MS_HttpRequestMessage"]));
```

Comment:
You kind of forget about this one: https://github.com/HanaNavratilova/kentico-onboarding-cs/pull/3#discussion_r219759984 😜